app/src/main/java/com/github/sdp/mediato/data/DatabaseUtils.java

```diff
@@ -29,7 +31,7 @@ class DatabaseUtils {
 
     static final int PROFILE_PIC_MAX_SIZE = 1024 * 1024; //1 Megabyte
 
-    static final int DEFAULT_RADIUS = 100;
+    public static final int DEFAULT_RADIUS = 100;

```

Comment:
You could add a comment explaining what this radius is used for and in which unit it is.

Reply:
Done